### ISSUE TITLE
About the training results

### ISSUE THREAD

**Issue by si520** [Ruifeng-Tan/BatteryLife#9]
Hi Tan , thank you for your great work！I have two questions to ask：
Q1: I have reproduced your code and here are the training results I obtained using CPMLP.sh on the MIX_large dataset. It seems that the validation loss is abnormal. Could you give me some advice? I would greatly appreciate it if you could share the best training parameters.

<img width="1062" height="463" alt="Image" src="https://github.com/user-attachments/assets/02efdcfc-a89b-4811-947c-5390d98c05d3" />

<img width="1060" height="255" alt="Image" src="https://github.com/user-attachments/assets/5c042c65-f0cb-4c2e-8b05-2d7791393d4c" />

Q2: Have you tried training on a single dataset (e.g., Tongji, XJTU)? I attempted to train the model on a single dataset, but the 15%-Acc and MAPE are still very low.

**Comment by Ruifeng-Tan** [comment]
Thank you for your question! Please find the point-by-point response below:
A1: The figure below shows the MAPE for the aging conditions contained in the validation set. We find that the MAPE for one aging condition (ISU-ILCC_G49C1.pkl) is extremely high, and we still do not clearly know the reason. The processed capacity, voltage, and current time series for this battery are still normal. Weixiang has updated the training hyperparameters for the CPMLP model, and you should obtain similar results by tuning the model around the given hyperparameters.

A2: Yes, we have tried training on a single dataset. The testing MAPE for XJTU and Tongji datasets should be ~8% and ~13%, respectively. However, it should be noted that the MAPE on datasets like MICH_EXP and ISU_ILCC is much higher. By the way, the 15%-Acc and MAPE cannot be low at the same time. It should be "high 15%-Acc and low MAPE" or "low 15%-Acc and high MAPE".


![Image](https://github.com/user-attachments/assets/54026bb9-bdc0-4f60-b339-dc0736ab7a43)

**Comment by si520** [comment]
Thank you for your reply. I have resolved the issue. Thank you for your open-source work.
